src/openfl/filters/GlowFilter.hx

```diff
@@ -353,18 +354,15 @@ private class GlowShader extends Shader {
 	public function new () {
 		
 		super ();
-		
-		#if !macro
-		data.uRadius.value = [ 0, 0 ];
-		data.uColor.value = [ 0, 0, 0, 0 ];
-		#end
-		
+				
 	}
 	
 	
 	private override function __update ():Void {
 		
-		data.uTextureSize.value = [ data.uImage0.input.width, data.uImage0.input.height ];
+		var u:ShaderParameterFloat2 = cast data.uTextureSize;
```

Comment:
Would be nice if there is a more explicit name of the parameter than just `u`. :)